x/ibc-hooks/contracts/crosschain-swaps/README.md

```diff
@@ -0,0 +1,214 @@
+# Crosschain Swaps
```

Comment:
Nice job on this! I think we should link to the counterparty chain outpost contract in readme as well (once that has its own place, etc)

Reply:
yes! once they... exist 😓